plugin/bufsize/setup.go

```diff
@@ -0,0 +1,52 @@
+package bufsize
+
+import (
+	"strconv"
+
+	"github.com/coredns/coredns/core/dnsserver"
+	"github.com/coredns/coredns/plugin"
+
+	"github.com/caddyserver/caddy"
+)
+
+func init() { plugin.Register("bufsize", setup) }
+
+func setup(c *caddy.Controller) error {
+	bufsize, err := bufsizeParse(c)
+	if err != nil {
+		return plugin.Error("bufsize", err)
+	}
+
+	dnsserver.GetConfig(c).AddPlugin(func(next plugin.Handler) plugin.Handler {
+		return Bufsize{Next: next, Size: bufsize}
+	})
+
+	return nil
+}
+
+func bufsizeParse(c *caddy.Controller) (int, error) {
+	const defaultBufSize = 512
```

Comment:
this const is only use once - just use the 512 value (or whatever we come up with later) directly

Reply:
Thank you for the comment.
This const is from https://github.com/coredns/coredns/pull/3401#discussion_r340229081 .
Which one is better?

Reply:
just leave it then, no reason to ping-ping between reviewers and @dilyevsky knows his stuff as well.

Reply:
Thanks! ok I'll leave it.